src/server/schemas/txOverrides.ts

```diff
@@ -9,16 +10,24 @@ export const txOverridesSchema = Type.Object({
           description: "Gas limit for the transaction",
         }),
       ),
-      maxFeePerGas: Type.Optional(
-        Type.String({
-          examples: ["1000000000"],
-          description: "Maximum fee per gas",
-        }),
-      ),
-      maxPriorityFeePerGas: Type.Optional(
-        Type.String({
-          examples: ["1000000000"],
-          description: "Maximum priority fee per gas",
+
+      // Overriding `gasPrice` is currently not supported.
+
+      maxFeePerGas: Type.Optional({
+        ...WeiAmountStringSchema,
+        description: "Maximum fee per gas",
+      }),
+      maxPriorityFeePerGas: Type.Optional({
+        ...WeiAmountStringSchema,
+        description: "Maximum priority fee per gas",
+      }),
+      timeoutSeconds: Type.Optional(
+        Type.Integer({
+          examples: ["7200"],
+          description:
+            "Maximum duration that a transaction is valid. If a transaction cannot be sent before the timeout, the transaction will be set to 'errored'. Default: no timeout",
+          minimum: 30,
```

Comment:
does it make sense to have a minimum is set to 30s timeout when the default is no timeout?